src/content/en/fundamentals/payments/payment-apps-developer-guide/android-payment-apps.md

```diff
@@ -289,6 +289,16 @@ choose a different payment app. The merchant website does not observe this, so
 there’s no need for detailed error codes to be sent from the payment app to the
 merchant website.
 
+Note: In Chrome 84 or later, if the activity result of a payment response
+received from the invoked payment app is set to `RESULT_OK`, then Chrome will
+check for non-empty `methodName` and `details` in its extras. If the validation
+fails Chrome will reject the `request.show()` promise with one of the following
+developer facing error messages:
```

Comment:
```suggestion
Note: In Chrome 84 and later, if the activity result of a payment response
received from the invoked payment app is `RESULT_OK`, then Chrome will
check for non-empty `methodName` and `details` in its extras. If the validation
fails Chrome will return a rejected promise from `request.show()` with one of the following
developer facing error messages:
```

I think this is more clear, but I'm not 100% sure.

Reply:
I think this is fine except the first `or`.